### ISSUE TITLE
PomBuilder - specify language level

### ISSUE THREAD

**Issue by hrstoyanov** [rife2/bld#39]
Hi,
when generating POM.xml file, for example:
```java
@BuildCommand(value = "pom-generate", summary = "Generates the POM file in the root directory")
    public void pomRoot() throws FileUtilsErrorException {
        PomBuilder.generateInto(publishOperation().fromProject(this).info(), dependencies(),new File(workDirectory, "pom.xml"));
    }
```

how can I specify the javac language leve, so I can have something like this in the generated POM:
```xml
<project>
  ...
  <build>
    <plugins>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.8.1</version>
        <configuration>
          <source>1.8</source>
          <target>1.8</target>
        </configuration>
      </plugin>
    </plugins>
  </build>
  ...
</project>
```

**Comment by gbevin** [comment]
What do you need this for?

**Comment by gbevin** [comment]
I'm doubtful that channeling plugin support through pom generation will ever be able to properly support bld. In any case, I have absolutely no plan to support any ideas besides IDEA and VSCode. The right way of doing this is working on a proper IDE plugin that is optimized for bld's nature and features.

**Comment by ethauvin** [comment]
> Also, I noticed that `PomBuilder.java`, lines 118-122 only outputs **compile** and **runtime** scopes, but not **provided**:
> 

That's definitely an oversight on our part.

**Comment by ethauvin** [comment]
I can understand trying to make it work with Oracle VS Code, but it should work out of the box with IDEA and RedHat VS Code if you use `./bld create`.

Oracle VS Code doesn't provide a way to configure a project other than through a Maven POM, which as @gbevin indicated is a non-starter for us.

I looked at supporting Eclipse and NetBeans a while back, and we decided that there wasn't enough interest to justify it. If I recall right, I was able to get it kinda working using `.project` and `.workspace` in Eclipse.
